reflectapi-demo/Cargo.toml

```diff
@@ -26,14 +26,14 @@ chrono = { version = "0.4.37", features = ["serde"] }
 bytes = "1.5.0"
 serde = { version = "1.0.197", features = ["derive", "rc"] }
 serde_json = "1.0.114"
+anyhow = "1.0.81"
```

Comment:
is it possible to keep the demo without this dependency? no particularly strong concern, but I tried to keep the demo as much as possible with little dependencies for others to understand it quicker even if they are not familiar with RUst ecosystem libs very well

Reply:
Switched to `Box<dyn Error>`. We should probably also consider moving most of the tests out of `demo` into `reflectapi`.